dataextractor.py

```diff
@@ -0,0 +1,300 @@
+#Importing
+from rdkit import Chem, RDConfig, Geometry
+from rdkit.Chem import AllChem, TorsionFingerprints
+import os, sys
+import openeye.oechem as oechem
+from openeye import oeshape
+import pandas as pd
+import numpy as np
+import itertools
+
+
+def tanimotocombo(ref_mol, query_mol):
+    """This is the TanimotoCombo function. It takes in two OEMols. 
+    It does not matter (for our purposes) which mol is the reference mol and 
+    which one is the query mol. The TanimotoCombo distance between the two
+    mols is the same no matter which is the reference and which is the query.
+    The GetTanimotoCombo() function comes from openeye. 
+    """
```

Comment:
add parameters and returns to all functions

Reply:
You should have something in your README.md which explains what these scripts are/what they are used for/how they are used.